saliency/blur_ig.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities to compute Blur IG SaliencyMask.
```

Comment:
add a copyright 2020 google clause here and the other file. See https://github.com/PAIR-code/saliency/blob/master/saliency/integrated_gradients.py#L1 for template

Reply:
Done.